src/components/editor/Bindings/SchemaEdit/Commands/ExistingSchema.tsx

```diff
@@ -0,0 +1,75 @@
+import { Typography } from '@mui/material';
+import SingleLineCode from 'components/content/SingleLineCode';
+import EditCommandsHeader from 'components/editor/Bindings/SchemaEdit/Commands/Header';
+import UpdateSchemaButton from 'components/editor/Bindings/SchemaEdit/Commands/UpdateSchemaButton';
+import {
+    useEditorStore_id,
+    useEditorStore_persistedDraftId,
+} from 'components/editor/Store/hooks';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useResourceConfig_currentCollection } from 'stores/ResourceConfig/hooks';
+
+function ExistingSchemaCommands() {
+    const intl = useIntl();
+
+    // Draft Editor Store
+    const draftId = useEditorStore_id();
+    const persistedDraftId = useEditorStore_persistedDraftId();
+
+    // Resource Config Store
+    const currentCollection = useResourceConfig_currentCollection();
+
+    const pullDraftCommands = {
```

Comment:
Any specific reason for an object vs an array here?

Reply:
The _hope_ was to reinforce that the order of the messages in this constant matter (e.g., the first command should always correspond to the first index, etc.). The `SingleLineCode` component seems like one that may be used in a map in the future. This was an experiment to get a sense of the most scalable way to bundle messages while communicating the importance of their order.

Arrays have a greater likelihood of being extended haphazardly; they are often prepended or appended when needed to be extended manually as opposed to programmatically. If order does matter, it is often expected that a utility function will enforce the desired order -- even in cases of manual extension. Consequently, a numerically-keyed object felt like the safest option.